scripts/dependabot_helper.bash

```diff
@@ -1,78 +1,18 @@
 #!/usr/bin/env bash
 
-# Dependabot is enabled for KEB, however, after making pre-main-kcp-cli check run for KEB as part of 
-# https://github.com/kyma-project/test-infra/pull/5776, the bot is struggling to produce PRs that 
-# would pass the pre-main-kcp-cli check. The discussion about long term path is currently under
-# https://github.com/kyma-project/control-plane/issues/1929 and this script is intended as a temporary
-# remediation until the consensus what to do in future is reached.
```

Comment:
this was initially the main reason but after https://github.com/kyma-project/control-plane/pull/2326, the script remains useful purely because there appear to continue to be minor challenges to enable automerge on dependabot PRs and the script is helpful in managing dependabot PRs efficiently.